src/main/java/seedu/address/logic/commands/UndoCommand.java

```diff
@@ -11,6 +11,8 @@
 public class UndoCommand extends Command {
     public static final String COMMAND_WORD = "undo";
     public static final String MESSAGE_NO_COMMAND_TO_UNDO = "There are no commands to undo.";
+    public static final String MESSAGE_NO_PREVIOUS_ADDRESS_BOOK_STATE =
+            "There is no saved history of TAPA to revert to.";
```

Comment:
Would this be considered a behavioural change or just a cosmetic one? I feel the teaching team is kind of spotty when it comes to error messages, and the previous error message is technically correct as well.

Reply:
@medajet Oh shoot, you're right, I did overlook this. I'll look through my code again and get back to you when I correct this!